src/transformers/models/deberta/modeling_tf_deberta.py

```diff
@@ -773,7 +773,8 @@ def call(
         Returns:
             final_embeddings (`tf.Tensor`): output embedding tensor.
         """
-        assert not (input_ids is None and inputs_embeds is None)
+        if input_ids is None and inputs_embeds is None:
+            raise ValueError("input_ids and input_embeds can not both be None")
```

Comment:
We usually have the same error message everywhere:
```suggestion
            raise ValueError("Need to provide either `input_ids` or `input_embeds`.")
```

Reply:
done!